storybooks/native/stories/Input.stories.js

```diff
@@ -15,6 +15,9 @@ storiesOf('Input/TextInput', module)
   .add('Default', () => {
     const placeholder = text('Placeholder', 'placeholder');
     const focus = boolean('Focus', false);
+    const autoFocus = boolean('AutoFocus', false);
```

Comment:
Didn't get this autoFocus-knob working via storybook, but when you do it like this it works: 
`<TextInput placeholder={placeholder} autoFocus focus={focus} error={error} />`

I think it's because there's delay before the knob values are added to the component, and autoFocus needs to be set true when component mounts

Reply:
Maybe there should be autoFocus set in a textInput/focus -story like in web storybook.
Also I'd add autoFocus prop in the same way to the story in web storybook

Reply:
- Made separate story for autofocus.
- Removed the whole focus knob
  - initializing focus state with autoFocus prop